client/me/concierge/shared/closure-notice.js

```diff
@@ -5,34 +5,39 @@
  */
 
 import React from 'react';
-import i18n, { localize } from 'i18n-calypso';
+import { useTranslate } from 'i18n-calypso';
+import 'moment-timezone'; // monkey patches the existing moment.js
```

Comment:
I really dislike the way `moment` and `moment-timezone` do things. Monkey-patching another import that isn't even explicitly in scope but is actually a global... Sigh.

The sooner we can get away from these libraries, the better, and the work you're doing here to move in that direction is more than welcome 👍